docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,284 @@ have not been included. These are:
+  * - Unit Number for any Address (Home OR Mailing OR Employer) 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 5%
+    - Miswriting
+    - Noise for all types of addresses will work in the same way 
+  * - PO Box for Mailing Address 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 5%
+    - Miswriting
+    - 
+  * - City Name for any Address (Home OR Mailing OR Employer) 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 5%
+    - OCR, phonetic, typographic
+    - Noise for all types of addresses will work in the same way 
+  * - State for any Address (Home OR Mailing OR Employer) 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 5%
+    - Incorrect Select
+    - Noise for all types of addresses will work in the same way 
+  * - Zip Code 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 5%
+    - Miswriting
+    - Applies to home, mailing, and employer addresses 
+  * - Relationship to head of household 
+    - Census 
+    - 5%
+    - Incorrect select 
+    - 
+  * - Sex 
+    - Census 
+    - 5%
+    - Incorrect select 
+    - 
+  * - Race/Ethnicity 
+    - Census, WIC
+    - 5%
+    - Incorrect select 
+    - 
+  * - SSN
+    - Taxes (both), SSA
+    - 5%
+    - Miswriting, "borrowed" SSN, Copy from within Household
+    - Note that not all types of noise apply to all observers, details below 
+  * - ITIN
+    - Taxes 1040
+    - 5%
+    - Miswriting, Copy from within Household
+    - Note that not all types of noise apply to all observers 
+  * - Income / Wages
+    - Taxes (both)
+    - 5%
+    - Miswriting
+    - Note that wages and income are on separate tax forms and noise is applied to each separately 
+  * - Employer ID 
+    - Taxes (both)
+    - 5%
+    - Miswriting
+    - 
+  * - Employer Name 
+    - Taxes (both)
+    - 5%
+    - OCR, typographic 
+    - 
+  * - Type of Tax Form  
+    - Taxes (both)
+    - 5%
+    - Incorrect select 
+    - 
+  * - Type of SSA Event 
+    - SSA 
+    - 5%
+    - Incorrect select 
+    - 
+  * - Date of SSA Event 
+    - SSA 
+    - 5%
+    - Miswriting, month and day swap 
+    - 
+
+The below section further describes types of noise including any code 
+available and information for implimentation. **Software engineering team - please alert the research team if any of the below looks to be particularly challenging for further discussion.**
+
+**OCR**
+
+Optical character recognition is when a character is misread for another character that 
+looks similar. A common examples is 'S' and '5'. In order to emulate 
+that, there is a `GeCo like corrupter and related list of possible changes in the ocr_variations_upper_lower csv found here <https://github.com/ihmeuw/vivarium_research_prl/tree/main/noise>`_. 
+
+Currently, the user defined rate will be a rate 'eligible' for noise. The actual rate of noise 
+will be determined at the character level as XX errors per character. We think this approximates to 1 
```

Comment:
>It'd be a bit cleaner in my opinion to ask the pseudopeople user to supply a per-character error probability, but it would make everything a bit more complicated.

@zmbc Are you suggesting having the end user supply *only* a per-character probability, e.g., applying to all characters in the column (or dataset?), *instead* of having them supply a row-eligibility probability? Or are you suggesting having them supply a per-character probability *in addition* to the row probability, so that it would represent the conditional probability of a character being corrupted, given that it is in a row that is selected?

Reply:
@NathanielBlairStahn good question. I was saying that it would be cleaner to have them _only_ supply a per-character probability. But, I get it if we don't want to do that.

On the other hand, having the option to customize the per-character probability conditional on the row being selected sounds like a good idea, if it isn't too hard to implement.